code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -539,6 +539,13 @@ to_chat will check for valid clients itself already so no need to double check f
 		var/mob/living/carbon/xenomorph/X = i
 		X.receive_hivemind_message(sender, message)
 
+///Used for setting the trackers of all xenos in the hive, like when a nuke activates
+/datum/hive_status/proc/set_all_trackers(atom/target)
```

Comment:
specify what is being set ie set_all_xenos_trackers

Reply:
as it is a member of `/datum/hive_status` and the autodoc says as much I don't see the need, but if required for approval I'll make it so